lightbulb/di.py

```diff
@@ -56,6 +56,8 @@
 """Injectable type representing the dependency container for the command context."""
 AutocompleteContainer = t.NewType("AutocompleteContainer", linkd.Container)
 """Injectable type representing the dependency container for the autocomplete context."""
+OptionConversionContainer = t.NewType("OptionConversionContainer", linkd.Container)
+"""Injectable type representing the dependency container for the option conversion context."""
```

Comment:
This is not necessary, option conversion will happen within the command context. This should be removed